hist/hist/src/TSpline.cxx

```diff
@@ -555,7 +555,7 @@ TSpline3& TSpline3::operator=(const TSpline3& sp3)
 {
    if(this!=&sp3) {
       TSpline::operator=(sp3);
-      fPoly= nullptr;
```

Comment:
To reduce duplicated code we could implement the assignment operator using the copy constructor

Reply:
Yes, I agree. This is quite old code though, and given that the copy-ctor does not suffer from the issue, I would not modify anything there, if you agree.